### PR TITLE
Create task-hierarchy.md in doc--In progress

### PR DESCRIPTION
**Pull request by swarooprm**

### CODE REVIEW

File: doc/Task Hierarchy.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+* Public relations
+* Security  studies
+* Sociology
+* Us foreign policy
+* Virology
+* World religions
+* Social Media
+
+Language
+- https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: refer to the names columns

File: doc/Task Hierarchy.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
+Task Type
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: rename the file so it doesn't have space in it + all lower case.

File: doc/Task Hierarchy.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+- Answer Generation: Open book/Domain-specific Question Answering
+- Answer Generation: Commonsense Question Answering
+- Answer Generation: Fill in the Blank
+- Textual Entailment
+- Incorrect Answer Generation
+- Sentence Generation
+- Text Modification
+- Classification
+- Paragraph Generation
+- Verification
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: define this

File: doc/Task Hierarchy.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+- Dialogue Understanding
+- Abuse Detection
+- Weblink Generation
+- Text Simplification
+- Reasoning: Numerical Reasoning
+- Coreference: Entity Coreference
+- Reasoning: Reasoning on Actions
+- Reasoning: General Commonsense Reasoning
+- Reasoning: Abductive Reasoning
+- Coreference: Pronoun Disambiguation
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Put the relevant categories together (coreferences, text generations, etc)

File: doc/Task Hierarchy.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+- Hate Speech Detection
+- Reasoning: Logic
+- Fake News Detection
+- Explanation Generation
+- Ethical Judgement
+- Style Transfer
+- Text Comparison
+- Question Composition
+- Review Generation
+- Column Matching
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define

File: doc/Task Hierarchy.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+- Analogical Reasoning
+- Semantic Parsing
+- Stance Detection
+- Word Sense Disambiguation
+- Topic Generation
+- Mathematics: Arithmetic
+- Mathematics: Algebra
+- Mathematics: Geometry
+- Question Decomposition
+- Counting
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define or exemplify

File: doc/Task Hierarchy.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+- Answer Generation: Fill in the Blank
+- Textual Entailment
+- Incorrect Answer Generation
+- Sentence Generation
+- Text Modification
+- Classification
+- Paragraph Generation
+- Verification
+- Translation
+- Multiple Choice Question Answering
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Put all QA's next to each other

File: doc/Task Hierarchy.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+- Summarization
+- Sentiment Analysis
+- Hate Speech Detection
+- Reasoning: Logic
+- Fake News Detection
+- Explanation Generation
+- Ethical Judgement
+- Style Transfer
+- Text Comparison
+- Question Composition
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define or exemplify. 
How is this different from QG?

File: doc/Task Hierarchy.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+- Textual Entailment
+- Incorrect Answer Generation
+- Sentence Generation
+- Text Modification
+- Classification
+- Paragraph Generation
+- Verification
+- Translation
+- Multiple Choice Question Answering
+- Structured Text Generation
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define or exemplify (what makes text generation "structured"?)

File: doc/Task Hierarchy.md [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+- Question Composition
+- Review Generation
+- Column Matching
+- Handling Commands
+- Author Identification
+- Order Generation
+- Language Identification
+- Prompt Generation
+- Text Span Selection
+- Hallucination
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define

File: doc/Task Hierarchy.md [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+- Explanation Generation
+- Ethical Judgement
+- Style Transfer
+- Text Comparison
+- Question Composition
+- Review Generation
+- Column Matching
+- Handling Commands
+- Author Identification
+- Order Generation
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define

File: doc/Task Hierarchy.md [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+- Coreference: Entity Coreference
+- Reasoning: Reasoning on Actions
+- Reasoning: General Commonsense Reasoning
+- Reasoning: Abductive Reasoning
+- Coreference: Pronoun Disambiguation
+- Reasoning: Temporal Reasoning
+- Reasoning: Spatial Reasoning
+- Reasoning: Multihop Reasoning
+- Reasoning: Reasoning on Social Interactions
+- Reasoning: Commonsense Reasoning
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define

File: doc/Task Hierarchy.md [hunk 13]
```diff
@@ -0,0 +1,165 @@
...
+- Reasoning: Numerical Commonsense Reasoning
+- Reasoning: Physical Reasoning
+- Reasoning: Logical Reasoning
+- Reasoning: Planning
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define

File: doc/Task Hierarchy.md [hunk 14]
```diff
@@ -0,0 +1,165 @@
...
+- Reasoning: General Commonsense Reasoning
+- Reasoning: Abductive Reasoning
+- Coreference: Pronoun Disambiguation
+- Reasoning: Temporal Reasoning
+- Reasoning: Spatial Reasoning
+- Reasoning: Multihop Reasoning
+- Reasoning: Reasoning on Social Interactions
+- Reasoning: Commonsense Reasoning
+- Reasoning: Numerical Commonsense Reasoning
+- Reasoning: Physical Reasoning
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define

File: doc/Task Hierarchy.md [hunk 15]
```diff
@@ -0,0 +1,165 @@
...
+- Reasoning: Reasoning on Actions
+- Reasoning: General Commonsense Reasoning
+- Reasoning: Abductive Reasoning
+- Coreference: Pronoun Disambiguation
+- Reasoning: Temporal Reasoning
+- Reasoning: Spatial Reasoning
+- Reasoning: Multihop Reasoning
+- Reasoning: Reasoning on Social Interactions
+- Reasoning: Commonsense Reasoning
+- Reasoning: Numerical Commonsense Reasoning
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define. (how is "numerical commensense" different from "numerical"?)

File: doc/Task Hierarchy.md [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+- Reasoning: Physical Reasoning
+- Reasoning: Logical Reasoning
+- Reasoning: Planning
+- Reasoning: Qualitative Reasoning
+- Reasoning: Deductive Reasoning
+- Reasoning: Causal Reasoning
+- Reasoning: Inverse-Causal Reasoning
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: I would merge cause and inverse-causal. They seem to be using the same principles.

File: doc/Task Hierarchy.md [hunk 17]
```diff
@@ -0,0 +1,165 @@
...
+- Reasoning: Reasoning with Symbols
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Define

File: doc/Task Hierarchy.md [hunk 18]
```diff
@@ -0,0 +1,165 @@
...
+- Question Decomposition
+- Counting
+- Entity Detection
+- Role Labelling
+- Combinatorics
+- Grammar Error Correction
+- Summarization
+- Sentiment Analysis
+- Hate Speech Detection
+- Reasoning: Logic
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Group all "Reasonings" together

File: doc/Task Hierarchy.md [hunk 19]
```diff
@@ -0,0 +1,165 @@
...
+* High school microeconomics
+* High school physics
+* High school psychology
+* High school statistics
+* High school us history
+* High school world history
+* Human aging
+* Human sexuality
+* International law
+* jurisprudence
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Uppercase

File: doc/Task Hierarchy.md [hunk 20]
```diff
@@ -0,0 +1,165 @@
...
+* Logical fallacies
+* Machine learning
+* Management
+* Marketing
+* Medical genetics
+* Miscellaneous
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Drop

File: doc/Task Hierarchy.md [hunk 21]
```diff
@@ -0,0 +1,165 @@
...
+- Reasoning: Logical Reasoning
+- Reasoning: Planning
+- Reasoning: Qualitative Reasoning
+- Reasoning: Deductive Reasoning
+- Reasoning: Causal Reasoning
+- Reasoning: Inverse-Causal Reasoning
+- Reasoning: Reasoning with Symbols
+- Tabular Text Operation
+
+Domain:
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Sort this alphabetically

File: doc/Task Hierarchy.md [hunk 22]
```diff
@@ -0,0 +1,165 @@
...
+* High school physics
+* High school psychology
+* High school statistics
+* High school us history
+* High school world history
+* Human aging
+* Human sexuality
+* International law
+* jurisprudence
+* Logical fallacies
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: "Logical fallacies"  vs "formal logic"?

File: doc/Task Hierarchy.md [hunk 23]
```diff
@@ -0,0 +1,165 @@
...
+* Machine learning
+* Management
+* Marketing
+* Medical genetics
+* Miscellaneous
+* Moral disputes
+* Moral scenarios
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: "Moral disputes" can be merged into "Moral scenarios"?

File: doc/Task Hierarchy.md [hunk 24]
```diff
@@ -0,0 +1,165 @@
...
+* High school government and politics
+* High school macroeconomics
+* High school mathematics
+* High school microeconomics
+* High school physics
+* High school psychology
+* High school statistics
+* High school us history
+* High school world history
+* Human aging
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: "Human aging" as a domain, sounds a bit odd.

File: doc/Task Hierarchy.md [hunk 25]
```diff
@@ -0,0 +1,165 @@
...
+* High school geography
+* High school government and politics
+* High school macroeconomics
+* High school mathematics
+* High school microeconomics
+* High school physics
+* High school psychology
+* High school statistics
+* High school us history
+* High school world history
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Merge the above two as "High school history"?

File: doc/Task Hierarchy.md [hunk 26]
```diff
@@ -0,0 +1,165 @@
...
+* Global facts
+* High school biology
+* High school chemistry
+* High school computer science
+* High school european history
+* High school geography
+* High school government and politics
+* High school macroeconomics
+* High school mathematics
+* High school microeconomics
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: repeated

File: doc/Task Hierarchy.md [hunk 27]
```diff
@@ -0,0 +1,165 @@
...
+* News
+* Society 
+* Law
+* Justice 
+* Anthropology
+* 9/11 reports
+* School science textbooks
+* Fiction
+* Personal narratives
+* National Football League (NFL) game summaries
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Maybe create a category for "Sports" and also change this to "Sports: NFL"?

File: doc/Task Hierarchy.md [hunk 28]
```diff
@@ -0,0 +1,165 @@
...
+* Business ethics
+* Clinical knowledge
+* College biology
+* College chemistry
+* College computer science
+* College mathematics
+* College medicine
+* College physics
+* Computer security
+* Conceptual physics
```
Comment (danyaljj) [inline doc/Task Hierarchy.md]: Not sure what this means

File: doc/task-hierarchy.md [hunk 29]
```diff
@@ -0,0 +1,176 @@
...
+* High school statistics
+* High school history
+* History
+* Human aging
+* Human sexuality
+* International law
+* Jurisprudence
+* Justice
+* Law
+* Law
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: Repeated

File: doc/task-hierarchy.md [hunk 30]
```diff
@@ -0,0 +1,176 @@
...
+- Sentence Generation
+- Paragraph Generation
+- Structured Text Generation
+   - Generate structured text in the output e.g. a task that converts questions in natural language to SQL queries
+- Topic Generation
+- Explanation Generation
+- Review Generation
+- Order Generation
+  - Given a set of elements, find their order (e.g. monotonically increasing/decreasing numbers, increasing/decreasing size in case of objects)
+- Prompt Generation
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: Define

Comment (danyaljj) [reply doc/task-hierarchy.md]: (how is different from summary or title generation?)

**Comment by danyaljj** [comment]
A few more comments added. Merge after addressing them.

File: README.md [hunk 31]
```diff
@@ -5,6 +5,8 @@ We're looking for more contributions to make this data bigger! 🙌
 We invite submission of new tasks to this benchmark by way of [GitHub pull request](https://github.com/allenai/natural-instructions-expansion/pulls), through **October 15, 2021**.
 The contributors with [meaningful contributions](https://github.com/allenai/natural-instructions-expansion/#what-do-you-mean-by-meaningful-contribution) to our tasks will be included as co-authors on a paper that will announce the benchmark as well as analysis/results on it. 
 
+## Summary of Diversity in the Accepted Tasks:
+Our tasks cover 86 task types, 38 languages and 74 domains. More details [here](doc/task-hierarchy.md).
```
Comment (danyaljj) [inline README.md]: I would drop this for now. After October 15th.

File: doc/task-hierarchy.md [hunk 32]
```diff
@@ -0,0 +1,174 @@
...
+* School science textbooks
+* Security  studies
+* Social Media
+* Society
+* Sociology
+* Sports: National Football League (NFL) game summaries
+* Sports
+* Us foreign policy
+* Virology
+* World religions
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: -> `Religions`

File: doc/task-hierarchy.md [hunk 33]
```diff
@@ -0,0 +1,174 @@
...
+* High school european history
+* High school geography
+* High school government and politics
+* High school macroeconomics
+* High school mathematics
+* High school microeconomics
+* High school physics
+* High school psychology
+* High school statistics
+* High school history
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: I would actually merge "high-school" and "college" categories.

File: doc/task-hierarchy.md [hunk 34]
```diff
@@ -0,0 +1,174 @@
...
+* History
+* Human aging
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: How about abstracting this a bit: `Human biology`

File: doc/task-hierarchy.md [hunk 35]
```diff
@@ -0,0 +1,174 @@
...
+* Public relations
+* School science textbooks
+* Security  studies
+* Social Media
+* Society
+* Sociology
+* Sports: National Football League (NFL) game summaries
+* Sports
+* Us foreign policy
+* Virology
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: -> `Biology: Virology`

File: doc/task-hierarchy.md [hunk 36]
```diff
@@ -0,0 +1,174 @@
...
+* Personal narratives
+* Philosophy
+* Prehistory
+* Psychology
+* Public relations
+* School science textbooks
+* Security  studies
+* Social Media
+* Society
+* Sociology
```
Comment (danyaljj) [inline doc/task-hierarchy.md:152]: Retire `Society` in favor of `Sociology`? (or the other way around)

File: doc/task-hierarchy.md [hunk 37]
```diff
@@ -0,0 +1,174 @@
...
+* Music
+* News
+* Nutrition
+* Personal narratives
+* Philosophy
+* Prehistory
+* Psychology
+* Public relations
+* School science textbooks
+* Security  studies
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: Not sure what kind of security this is. Is this computer security? national security?

File: doc/task-hierarchy.md [hunk 38]
```diff
@@ -0,0 +1,164 @@
...
+* Public Relations
+* School Science Textbooks
+* Security: Environmental Security
+* Security: National Security 
+* Social Media
+* Sociology
+* Sports: National Football League (NFL) Game Summaries
+* Sports
+* Us Foreign Policy
+* Biology: Virology
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: Move up next to other "biology"

File: doc/task-hierarchy.md [hunk 39]
```diff
@@ -0,0 +1,164 @@
...
+- Reasoning: Logical Reasoning
+- Reasoning: Planning
+   - Tasks which need some sort of planning e.g. how to go to Hawaii?
+- Reasoning: Qualitative Reasoning
+- Reasoning: Deductive Reasoning
+- Reasoning: Causal Reasoning
+- Reasoning: Reasoning with Symbols
+   - Tasks where symbols represent various things e.g. if X is the number of apples in the freeze today morning and Y is the number remaining after I ate a few apples, X-Y is the number of apples I ate.
+
+Domain:
```
Comment (danyaljj) [inline doc/task-hierarchy.md:97]: Should "dialogue" also be a domain?

File: doc/task-hierarchy.md [hunk 40]
```diff
@@ -0,0 +1,164 @@
...
+* Microeconomics
+* Psychology
+* Statistics
+* Biology: Human Biology
+* Human Sexuality
+* International Law
+* Jurisprudence
+* Justice
+* Law
+* Logical Fallacies
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: "Logical Fallacies" sounds a bit niche. How can we make it a bit more general?

File: doc/task-hierarchy.md [hunk 41]
```diff
@@ -0,0 +1,164 @@
...
+* Machine Learning
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: -> `Computer Science: Machine Learning`

File: doc/task-hierarchy.md [hunk 42]
```diff
@@ -0,0 +1,164 @@
...
+* Geography
+* Global Facts
+* History
+* History: European History
+* Government and Politics
+* Macroeconomics
+* Microeconomics
+* Psychology
+* Statistics
+* Biology: Human Biology
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: sort

File: doc/task-hierarchy.md [hunk 43]
```diff
@@ -0,0 +1,164 @@
...
+- Reasoning: Qualitative Reasoning
+- Reasoning: Deductive Reasoning
+- Reasoning: Causal Reasoning
+- Reasoning: Reasoning with Symbols
+   - Tasks where symbols represent various things e.g. if X is the number of apples in the freeze today morning and Y is the number remaining after I ate a few apples, X-Y is the number of apples I ate.
+
+Domain:
+* 9/11 Reports
+* Accounting
+* Biology: Anatomy
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: sort

File: doc/task-hierarchy.md [hunk 44]
```diff
@@ -0,0 +1,164 @@
+Task Type
+- Contextual Question Generation
+   - Generate questions based on given context e.g. a task to create a question based on a paragraph.
+- Answer Generation: Reading Comprehension
+- Answer Generation: Contextual Question Answering
+- Answer Generation: Open book/Domain-specific Question Answering
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: What is "Domain-specific" QA?

File: doc/task-hierarchy.md [hunk 45]
```diff
@@ -0,0 +1,164 @@
+Task Type
+- Contextual Question Generation
```
Comment (danyaljj) [inline doc/task-hierarchy.md:2]: Add "Question Generation" prefix for all the QG variants.

File: doc/task-hierarchy.md [hunk 46]
```diff
@@ -0,0 +1,164 @@
...
+   - Given a context, generate imaginary content e.g. given a sentence, generate a story/poem.
+- Document Classification
+- Relevancy Estimation
+- Named Entity Recognition
+- Dialogue Understanding
+- Abuse Detection
+- Text Simplification
+- Tabular Text Operation
+- Coreference: Entity Coreference
+- Coreference: Pronoun Disambiguation
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: sort

File: doc/task-hierarchy.md (hunk 44 above)
Comment (danyaljj) [reply doc/task-hierarchy.md]: How about "Open QA" or "Direct-answer QA"?

File: doc/task-hierarchy.md [hunk 47]
```diff
@@ -0,0 +1,164 @@
+Task Type
+- Contextual Question Generation
+   - Generate questions based on given context e.g. a task to create a question based on a paragraph.
+- Answer Generation: Reading Comprehension
+- Answer Generation: Contextual Question Answering
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: What is the difference between "Contextual Question Answering" and "Reading Comprehension"?

File: doc/task-hierarchy.md [hunk 48]
```diff
@@ -0,0 +1,164 @@
...
+- Text Modification
+- Classification
+- Verification 
+   - Verify if two inputs match with each other e.g. a task to verify if a sentence and its sentiment given in the input match
+- Translation
+- Hypernym Discovery
+- Parts-of-speech
+- Coreference: event coreference
+- Intent Detection
+- Claim Verification
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: Is "Claim Verification" a subcategory of "Verification"?

File: doc/task-hierarchy.md (hunk 47 above)
Comment (swarooprm) [reply doc/task-hierarchy.md]: Context= passage in "Reading Comprehension"
Context= single sentence/word in "Contextual Question Answering"

File: doc/task-hierarchy.md (hunk 44 above)
Comment (swarooprm) [reply doc/task-hierarchy.md]: Open QA sounds good.

File: doc/task-hierarchy.md (hunk 48 above)
Comment (swarooprm) [reply doc/task-hierarchy.md]: Yes, will add header

File: doc/task-hierarchy.md (hunk 39 above)
Comment (swarooprm) [reply doc/task-hierarchy.md:97]: I think dialogue is too abstract to be a domain. Dialogue can be on various domains e.g. medical, movies, sports etc.

Also, we have a task type: "Dialogue Understanding".

File: doc/task-hierarchy.md (hunk 40 above)
Comment (swarooprm) [reply doc/task-hierarchy.md]: How about Formal Fallacy (https://www.google.com/search?q=Logical+Fallacies&oq=Logical+Fallacies&aqs=chrome..69i57.1392j0j7&sourceid=chrome&ie=UTF-8)?

File: doc/task-hierarchy.md (hunk 39 above)
Comment (danyaljj) [reply doc/task-hierarchy.md:97]: I would still suggest having a "Dialogue" as a text domain. The language of say, Alexa conversations are much different than anything else that we have on the list.

File: doc/task-hierarchy.md [hunk 49]
```diff
@@ -0,0 +1,164 @@
...
+- Text Span Selection
+- Hallucination
+   - Given a context, generate imaginary content e.g. given a sentence, generate a story/poem.
+- Document Classification
+- Relevancy Estimation
+- Named Entity Recognition
+- Dialogue Understanding
+- Abuse Detection
+- Text Simplification
+- Tabular Text Operation
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: "Tabular Text Operation" and "Column Matching" seem to be relevant. Should they have a common ancestor?

File: doc/task-hierarchy.md (hunk 39 above)
Comment (swarooprm) [reply doc/task-hierarchy.md:97]: Added

**Comment by danyaljj** [comment]
LGTM!

File: doc/task-hierarchy.md [hunk 50]
```diff
@@ -0,0 +1,160 @@
...
+- Dialogue Understanding
+- Document Understanding
+- Entity Detection
+- Ethical Judgement
+- Explanation Generation
+- Fake News Detection
+- Grammar Error Correction
+- Hallucination
+   - Given a context, generate imaginary content e.g. given a sentence, generate a story/poem.
+- Handling Commands
```
Comment (danyaljj) [inline doc/task-hierarchy.md]: What does "Handling" mean? Is handling == executing?

Comment (swarooprm) [reply doc/task-hierarchy.md]: Yes, changing it to executing.

**Pull request merged**